src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetRangeValueCreator.ts

```diff
@@ -14,7 +14,12 @@ export class DynamicFacetRangeValueCreator implements IValueCreator {
     this.parser = new DynamicFacetRangeValueParser(this.facet);
   }
 
-  public createFromRange(unvalidatedRange: IRangeValue, index: number) {
+  public getDefaultValues() {
+    const ranges = this.facet.options.ranges || [];
```

Comment:
Would it be better to change the `options.ranges` to have a default value of `[]` ?